src/main/kotlin/uk/gov/dwp/dataworks/services/TaskDeploymentService.kt

```diff
@@ -0,0 +1,136 @@
+package uk.gov.dwp.dataworks.services
```

Comment:
General comments:
- The `: <type>` is only needed in Kotlin where the type is ambiguous (and Kotlin will fail to compile if this is the case) so we can remove a lot of the definitions from the code.

Reply:
I can still see a couple of these flaoting around